.evergreen.yml

```diff
@@ -70,12 +89,21 @@ functions:
         add_to_path:
           - ${workdir}/bin
         working_dir: mongodb-kubernetes-operator
-        include_expansions_in_env:
-          - version_id
-          - test
-        binary: scripts/ci/run_test.sh
         env:
           KUBECONFIG: ${workdir}/kube_config
+        command: "pip3 install -r ./requirements.txt"
```

Comment:
I had to have this separated from the other one instead of:

`pip3 install -r ./requirements.txt && python ...` as evergreen was doing something weird and complaining about passing `--test` to pip.